history/characters/160000_quilboar.txt

```diff
@@ -2705,13 +2716,20 @@
 	636.11.7={ death=yes }
 }
 160274={
-	name=Baldaff
+	name=Lok
 	dynasty=112002
 	culture=quilboar religion=agamaggan_worship
 	martial=5 diplomacy=5 stewardship=6 intrigue=5 learning=5
 	trait=education_intrigue_4 trait=gluttonous trait=content 
 	father=160271	#Hin
-	566.9.12={ birth=yes trait=creature_quilboar }
+	559.5.10={ birth=yes trait=creature_quilboar }
+	583.1.1 = {
+		effect = {
+			set_variable = { name = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
+			set_variable = { name = wc_strength_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
+		}
+	}
+	603.1.1 = { give_nickname = nick_orcbane }
```

Comment:
It could be a founded house too

Reply:
It's said he's the apparent leader of Razormane, so sticking to that.